python/lsst/summit/utils/quickLook.py

```diff
@@ -233,20 +253,28 @@ def run(self, ccdExposure, *,
             self.log.info("Running with ptc correction")
 
         isrConfig.doWrite = False
-        isrTask = IsrTask(config=isrConfig)
+        isrTask = self.isrTask(config=isrConfig)
         result = isrTask.run(ccdExposure,
                              camera=camera,
                              bias=bias,
                              dark=dark,
                              flat=flat,
-                             #  fringes=pipeBase.Struct(fringes=None),
+                             fringes=fringes,
```

Comment:
Now that we're actually accepting the fringes if they're passed in, I think we should also have a block that tests if we got fringes supplied, and if so, sets the config option on to use them. We can then remove the `# TODO: deal with fringes here` comment too 🙂

Reply:
I think just replacing that TODO with
```
        if fringes.fringes is not None:
            isrConfig.doFringe = True
            self.log.info("Running with fringe correction")
```
should do it.